src/util.js

```diff
@@ -7,7 +7,15 @@
  * @return {*}
  */
 export function find (list, f) {
-  return list.filter(f)[0]
+  const { length } = list
```

Comment:
Can we revert this change? It looks not related with docs update.
Also, I'm not a fan of making the code longer for such micro optimization.